src/js/equity-dash/charts/disparity-multi-linechart/index.js

```diff
@@ -0,0 +1,295 @@
+import template from "./disparity-template.js";
+import getTranslations from "../../../common/get-strings-list.js";
+import getScreenResizeCharts from "../../../common/get-window-size.js";
+import rtlOverride from "../../../common/rtl-override.js";
+import chartConfig from './disparity-chart-config.json';
+import renderChart from "./disparity-chart.js";
+import termCheck from "../race-ethnicity-config.js";
+import applySubstitutions from "./../../../common/apply-substitutions.js";
+import { getSnowflakeStyleDate, reformatReadableDate } from "../../../common/readable-date.js";
+import formatValue from "./../../../common/value-formatters.js";
+import { hasURLSearchParam, getURLSearchParam}  from "./geturlparams.js";
```

Comment:
Could `src/js/dashboard/charts/common/geturlparams.js` have been imported here?

Reply:
This code will only execute on staging branch, so it's okay for now to leave the console.logs in.  In practice, we still have a few stray console.logs floating around.  I'll fix the typo and check the import.

Reply:
I didn't want to use dashboard/charts/ code because it's local to for the dashboard page. The proper fix is to move geturlparams to the src/js/common/ directory, and remove both of the more localized instances.  I will do so.

Reply:
Nice catches James!

Reply:
Great catches @JamesSLogan! We haven't been doing this on covid because we wrote so much code before we started doing it but on other projects now we use prettier and eslint with a custom config to run on local commit. The https://github.com/cagov/design-system/blob/main/package.json#L14 project is a good example